src/pocketmine/entity/Effect.php

```diff
@@ -251,14 +253,19 @@ public function setColor($r, $g, $b){
 	}
 
 	public function add(Entity $entity, $modify = false, Effect $oldEffect = null){
+		$entity->getLevel()->getServer()->getPluginManager()->callEvent($ev = new EntityEffectAddEvent
+		($entity, $this, $modify, $oldEffect));
```

Comment:
Accidently changed some settings in PHPStorm when I was messing around the other day, fixed.